ts-client/src/dlmm/index.ts

```diff
@@ -4284,6 +4286,177 @@ export class DLMM {
     };
   }
 
+  /**
+ * The `seedLiquidity` function create multiple grouped instructions. The grouped instructions will be either [initialize bin array + initialize position instructions] or [deposit instruction] combination.
+ * @param
+ *    - `owner`: The public key of the positions owner.
+ *    - `base`: Base key
+ *    - `seedAmount`: Lamport amount to be seeded to the pool.
+ *    - `isTokenX`: Indicate that the seed amount will be 100 percent token X or token Y.
+ *    - `price`: TokenX/TokenY Price in UI format
+ *    - `roundingUp`: Whether to round up the price
+ *    - `feeOwner`: Position fee owner
+ *    - `operator`: Operator of the position. Operator able to manage the position on behalf of the position owner. However, liquidity withdrawal issue by the operator can only send to the position owner.
+ *    - `lockReleasePoint`: The lock release point of the position.
+ * 
+ * The returned instructions need to be executed sequentially if it was separated into multiple transactions.
+ * @returns {Promise<TransactionInstruction[]>}
+ */
+  public async seedLiquiditySingleBin(
+    owner: PublicKey,
+    base: PublicKey,
+    seedAmount: BN,
+    isTokenX: boolean,
```

Comment:
because the purpose of endpoint is to seed liquidity for launchpool, so I think you can remove that flag
(I thought that is created for general purpose)

Reply:
i think it's okay to have that argument, i want to make the function as general as possible